Doc/tutorial/introduction.rst

```diff
@@ -195,7 +195,7 @@ an odd number of ``\`` characters; see
 and workarounds.
 
 String literals can span multiple lines.  One way is using triple-quotes:
-``"""..."""`` or ``'''...'''``.  End of lines are automatically
+``"""..."""`` or ``'''...'''``.  Ends of lines are automatically
```

Comment:
Hmmm, Why do I feel lowercase is more beautiful.
```
String literals can span multiple lines.  One way is using triple-quotes:
``"""..."""`` or ``'''...'''``.  ends of lines are automatically included in the string,
but it's possible to prevent this by adding a ``\`` at the end of the line. 
```